Build/Customizations/Server/Extensions.Server.targets

```diff
@@ -70,7 +70,6 @@
   <Target Name="Server_CIBuild_Chrome" DependsOnTargets="TestBuild_ChromeConfiguration;CheckProperties;SetVersionCiAndNightlyBuild;TestBuild" />
```

Comment:
I found the following code elements that should be removed too:
- `Remotion.Web.ExecutionEngine.WxePageInfo.c_styleFileUrlForIE`
- `Remotion.Web.ExecutionEngine.WxePageInfo.s_styleFileKeyForIE`

Reply:
Those are out of scope for the current task (webtests only)